.github/workflows/merge_queue.yml

```
@@ -0,0 +1,785 @@
+      base_repo_name: ${{ github.repo.repo }}
+      head_sha: ${{ github.event.merge_group.head_sha }}
+      head_repo_owner: ${{ github.repo.owner }}
+      head_repo_name: ${{ github.repo.repo }}
+      test_aws_role: ${{ steps.properties.outputs.test-aws-role || 'arn:aws:iam::799847381734:role/cptest-test-gbl-sandbox-gha-iam-terratest' }}
+      test_skip_concurrency: ${{ steps.properties.outputs.test-skip-concurrency }}
+      test_needs_aws: ${{ steps.properties.outputs.test-needs-aws }}
+      test_needs_fixtures: ${{ steps.properties.outputs.test-needs-fixtures }}
+      atmos_version: 1.129.0
+      uses_github: ${{ contains(github.repo.repo, '-github-') ||
+        contains(steps.properties.outputs.providers, 'github') }}
+      uses_opsgenie: ${{ contains(github.repo.repoe, '-opsgenie-') ||
```

actionlint (1.7.4)

property "repo" is not defined in object type {action: string; action_path: string; action_ref: string; action_repository: string; action_status: string; actor: string; actor_id: string; api_url: string; base_ref: string; env: string; event: object; event_name: string; event_path: string; graphql_url: string; head_ref: string; job: string; job_workflow_sha: string; path: string; ref: string; ref_name: string; ref_protected: string; ref_type: string; repository: string; repository_id: string; repository_owner: string; repository_owner_id: string; repositoryurl: string; retention_days: number; run_attempt: string; run_id: string; run_number: string; secret_source: string; server_url: string; sha: string; token: string; triggering_actor: string; workflow: string; workflow_ref: string; workflow_sha: string; workspace: string}

(expression)